addons/turbo-editor/userscript.js

```diff
@@ -0,0 +1,24 @@
+export default async function({ addon, global, console }) {
+    console.log("Turbowarp Scratch Mod (c) 2020 YOYITsM3M8 has loaded");
+    var current_url = document.location; // get url
+    var id = current_url.href.split('/')[4]; // get project id from url
+    function check() { // check if page loaded
+        const el = document.querySelector('.stage-wrapper_stage-wrapper_2bejr');
+        try {
+            if (el.classList[0] == "stage-wrapper_stage-wrapper_2bejr") {
+                el.removeChild(el.childNodes[0]);
+                el.removeChild(el.childNodes[0]);
+                var turboframe = document.createElement("iframe");
+                turboframe.src="https://turbowarp.org/fullscreen.html#"+id;
```

Comment:
instead of `https://turbowarp.org/fullscreen.html#`, you should use `https://turbowarp.org/embed.html#`

Reply:
:O didn’t know that existed

Reply:
Forkphorus also has that.